cmd/admin-handlers.go

```diff
@@ -801,6 +810,91 @@ func convertValueType(elem []byte, jsonType gjson.Type) (interface{}, error) {
 	}
 }
 
```

Comment:
this function needs a comment on the data format it is expecting and what would constitute an invalid field. Do also consider adding comments to specific lines that will make the intent of the code clear to the reader.

Reply:
and tests

Reply:
@poornas : OK

@harshavardhana : OK